lib/oelite/function.py

```diff
@@ -120,6 +121,28 @@ def __init__(self, meta, var, name=None, tmpdir=None):
         super(ShellFunction, self).__init__(meta, var, name, tmpdir)
         return
 
+    def runscript(self, cmd):
```

Comment:
It takes one command, which is the path to the script we've written to the .run file. That command is prepended with LC_ALL=C and possibly other stuff, which means that we still need the shell to interpret the command.